js/rooms.js

```diff
@@ -78,4 +104,18 @@ $(document).ready(function() {
 		}
 	};
 
+	$(document).click(function(e) {
```

Comment:
I'm not a fan of those global handlers but it *should* work as a first version :-)

(Note: I've implemented such a menu for Mail too, maybe we can re-use some code)